monit/config.sls

```diff
@@ -8,8 +8,9 @@
     - makedirs: True
     - mode: '0700'
     - context:
-        config_includes: {{ monit.config_includes | json }}
+        config_includes: {{ monit.config_includes }}
```

Comment:
why remove the ` | json` here?

Reply:
Oversight, or a mistake. You choose ;)
I've put it back now.